images/win/Windows2016-Readme.md

```diff
@@ -549,7 +549,7 @@ All other versions are saved but not installed.
 | Android SDK Tools          | 25.2.5                                                                                                                                                                                                                                                                                    |
 | Android Support Repository | 47.0.0                                                                                                                                                                                                                                                                                    |
 | CMake                      | 3.10.2<br>3.18.1                                                                                                                                                                                                                                                                          |
-| Google APIs                | addon-g..._apis-google-21<br>addon-g..._apis-google-22<br>addon-g..._apis-google-23<br>addon-g..._apis-google-24<br>a...google_apis                                                                                                                                                       |
+| Google APIs                | addon-google_apis-google-21<br>addon-google_apis-google-22<br>addon-google_apis-google-23<br>addon-google_apis-google-24<br>android-27                                                                                                                                                    |
```

Comment:
Looks like something went wrong here with android-27, I'll take a look at the software generator

Reply:
Fix https://github.com/actions/virtual-environments/pull/3715